tests/lib/tools/fs-state

```diff
@@ -77,6 +80,66 @@ restore_file() {
     fi
 }
 
+start_monitor() {
+    if [ -n "$(command -v inotifywait)" ]; then
+        # In systems like arch-linux the inotifywait command fails when the output file does not exist
+        touch /tmp/fs.output
+        touch /tmp/fs.exclude
+
+        # Monitor files and directories creation
+        SNAP_MOUNT_DIR="$(os.paths snap-mount-dir)"
+
+        # $PWD is excluded because the backup dir is automatically restored
+        # .snap files created in tests/lib/snaps are excluded as they are not deleted to be reused
+        inotifywait -d -m -r -e CREATE -o /tmp/fs.output --exclude "($PWD|$PROJECT_PATH/tests/lib/snaps/.*/.*.snap)" /root /var/lib/snapd /var/snap /home "$SNAP_MOUNT_DIR"
+        retry -n 10 -wait 1 sh -c 'pgrep inotifywait &>/dev/null'  
+    fi
+}
+
+check_monitor() {
+    if [ -n "$(command -v inotifywait)" ]; then
+        # Stop inotifywait (it could be already stopped)
+        if pgrep inotifywait &>/dev/null; then
+            pkill inotifywait
+        fi
```

Comment:
nitpick: I guess that with

    pkill inotifywait || true

you could get rid of the `if` block.

Reply:
That's a good idea, also maybe `pkill -9 inotifywait || true`

Reply:
inotifywait is not being used anymore